staging/src/k8s.io/apiserver/pkg/storage/cacher/time_budget.go

```diff
@@ -57,7 +62,7 @@ func newTimeBudget(stopCh <-chan struct{}) *timeBudget {
 	return result
```

Comment:
this is orthogonal to this issue, but I was surprised to see an active refresher process here... I expected something more like https://github.com/kubernetes/kubernetes/blob/master/vendor/golang.org/x/time/rate/rate.go#L306-L331 where the budget has an internal clock and computes budget on demand rather than waiting on an active background refresher process at 1-second intervals

Reply:
Filed https://github.com/kubernetes/kubernetes/issues/95875